### ISSUE TITLE
Automatically generate your SDKs

### ISSUE THREAD

**Issue by dsinghvi** [metaphorsystems/metaphor-go#3]
👋 @jeffzwang -- Stumbled on Metaphor on twitter and saw that you offer SDKs for your REST API. 

I work at [Fern](https://buildwithfern.com/) and we help companies automatically generate idiomatic SDKs from their OpenAPI spec. Fern handles reading your OpenAPI spec, generating source code, mirroring to GitHub and publishing to each registry. 

Checkout some examples of generated SDKs: 
- https://github.com/FlatFilers/flatfile-node
- https://github.com/merge-api/merge-go-client
- https://github.com/vellum-ai/vellum-client-python
- https://github.com/flipt-io/flipt-java

It looks like you're manually generating SDKs today. Is this something you would want to outsource?

**Comment by jeffzwang** [comment]
Hi! This would be great if the SDKs are of high quality. I unfortunately haven't had much luck with OpenAPI SDK generators as a whole. Is Fern of especially high quality?

**Comment by dsinghvi** [comment]
Yup, we have rebuilt SDK generators from scratch to improve the code generation quality. We currently support Python, TypeScript, Go and Java. Would you be up to hop on a zoom sometime? (Here's a link to my [calendly](https://calendly.com/deep-singhvi/30min?month=2023-08))
